VaticanBAV/et/BAVet122.xml

```diff
@@ -0,0 +1,453 @@
+                           <incipit xml:lang="gez">
+                              ወካዕበ፡ ናስተበቍዕ፡ ዘኵሎ፡ ይእኅዝ፡ እግዚአብሔር፡ አብ፡ <gap reason="omitted" resp="PRS4805Grebaut PRS9530Tisseran"/> እስመ፡ ረሰየ፡ ለነ፡ ተድላ፡
+                           </incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.4.6">
+                           <locus target="#18v"/>
+                           <title ref="LIT4263Prayer">ሊጦን፡ ዘአር<supplied reason="omitted">ብ</supplied>፡</title>
+                           <incipit xml:lang="gez">
+                              ናስተበቍዕ፡ ዘኵሎ፡ ይእኅዝ፡ እግዚአብሔር፡ አብ፡ <gap reason="omitted" resp="PRS4805Grebaut PRS9530Tisseran"/> እንዘ፡ <sic>ነአኵቶ፡</sic> በኵሉ፡ 
+                              ወውስተ፡ ኵሉ፡ እስመ፡ ከደነነ፡ ወረድአነ፨ ዓቀበነ፡ ወአብጽሐነ፡ እስከ፡ ዛቲ፡ ሰአት፡
+                           </incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.4.7">
+                           <locus from="18v" to="20v"/>
+                           <title ref="LIT1960Mashaf#BaendaEllaNomu4">ሊጦን፡ <choice><sic>ዘመታ፡</sic><corr resp="PRS4805Grebaut PRS9530Tisseran">ዘሙታን፡</corr></choice></title>
+                           <incipit xml:lang="gez">
+                              ወካዕበ፡ ናስተበቍዕ፡ በል፨ እግዚ<supplied reason="explanation">አ፡</supplied> ሕያዋን፡ ወሕይወተ፡ ሙታን፡ ተስፋ፡ ቅቡፃን፡ ረዳኤ፡ ምንዱባን፡ ወመንጽሔ፡ 
+                              ኃጥአን፡ ዘሞተ፡ አጽራዕከ፡ ወማዕሠረ፡ ሰይጣን፡ በቲከከ፡ ሕይወት፡ <sic>ለትዝምድ፡</sic> ሰብእ፡
+                           </incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.4.8">
+                           <locus from="20v" to="21r"/>
+                           <title ref="LIT1960Mashaf#SuppMasqal">ሊጦን፡ ዘመ<supplied reason="omitted">ስ</supplied>ቀል፨</title>
+                           <incipit xml:lang="gez">
+                              ወካዕ<supplied reason="omitted">በ</supplied>፡ ናስተበቍዕ፡ ለዕፀ፡ መስቀል፡ ቅዱስ፡ ጽኑዓ፡ ሥልጣን፡ ወኃይል፡ ወሊሉየ፡ መልክዕ፨
+                           </incipit>
+                        </msItem>
+                     </msItem>
+                     
+                     <msItem xml:id="ms_i1.5">
+                        <locus from="21r" to="50v"/>
+                        <title ref="LIT1960Mashaf#SerataQeddase">Ordinary of the Mass</title>                        
+                        <incipit xml:lang="gez">
+                           <locus target="#21r"/>
+                           በስመ፡ አብ፡ <gap reason="omitted" resp="PRS4805Grebaut PRS9530Tisseran"/> ንጽሕፍ፡ ሥርዓተ፡ ቅዳሴ፡ ከመ፡ ይበሉ፡ ቀሲስ፡ ወዲያቆን፡ ወሕዝብ፡ ወኵሉ፡ 
+                           ዘመፍትው፡ በበጊዜሁ፡ በከመ፡ ሥርዓተ፡ አበዊነ፡ ቅዱሳን፡
+                        </incipit>
+                        <note>The Ordinary of the Mass is separated from the anaphoras, like in recent manuscripts.
+                           The text bears a second title after the prayers for the blessing of the tools for the Eucharist:
+                           <q xml:lang="gez">
```

Comment:
@eu-genia I am not sure if q can be used outside of additions or decorations (I believed that not, but could not find it in the guidelines) or if foreign should be used, could you refresh my memory?

Reply:
I believe `q` is fine anywhere